lib/menu.rb

```diff
@@ -0,0 +1,29 @@
+class Menu
```

Comment:
I see a `Menu` but no `menu_spec.rb`. If you need a class, make sure you are driving it through Test-Driven Development!